README.md

```diff
@@ -89,7 +95,10 @@ After it finishes, you can find your images in the `output` directory on your ho
 
 
 ### Known issues
-* Raspberrypi0w cpu isn't armv7+ architecture (it's armv6) and because of this mender client + u-boot fw_set/getenv tools are crashing when compiled with armv7 toolchain added in docker image. Pls use this forked [repo](https://github.com/nandra/mender-conversion-tools/commits/rpi0w-toolchain) to have it properly build with other armv6 toolchain.
```

Comment:
We should re-word this and keep a note about RPi0W not working due to, https://tracker.mender.io/browse/MEN-2436

Reply:
Done.